### ISSUE TITLE
more arches for clang

### ISSUE THREAD

**Issue by nickdesaulniers** [kernelci/kernelci-core#517]
We should be able to build with clang:
1. malta_kvm_guest_defconfig for mips and mipsel
2. ppc44x_defconfig for powerpc (32b)
3. pseries_defconfig and powernv_defconfig for ppc64 (BE and LE respectively)
4. defconfig for riscv
5. defconfig for s390
6. defconfig for i386 / i386_defconfig

**Comment by gctucker** [comment]
Is this all to be built with clang 11 on linux-next?

(see my other question on #518 about mainline and the extend of the build coverage on each branch)

**Comment by broonie** [comment]
These all seem like reasonable targets to me, especially if we don't do any allmodconfigs which would consume lots of build resource. 

Might also be worth throwing allnoconfigs in there for the ones that it works on, they're tiny and do throw up issues from time to time.

**Comment by gctucker** [comment]
> Might also be worth throwing allnoconfigs in there for the ones that it works on, they're tiny and do throw up issues from time to time.

Yep it's there for riscv:
https://github.com/kernelci/kernelci-core/pull/534/files#diff-ec5c0ba0497e23f334c288a5ca3580e72c83e4bd1eac5a6627c33c22475cc2d2R300

**Comment by gctucker** [comment]
@nickdesaulniers I see you're now using `tuxbuild` with your own CI system, so are you still looking at kernelci.org results?

**Comment by nickdesaulniers** [comment]
Absolutely; I have to file a bunch of bugs in our issue tracker from the latest email report. It's on my TODO list.

My goal with the various kernel CI systems is redundancy.  Each service has its tradeoffs, and https://blog.travis-ci.com/oss-announcement blew up our initial CI so it's not given that any one service sticks around forever.

**Comment by gctucker** [comment]
OK great, thanks for confirming.  Please continue to provide feedback so we can keep improving things.  The KernelCI build tool is being redesigned now to become more flexible, there may be ways for Clang builds and your workflow to benefit from that.
